index.html

```diff
@@ -181,13 +180,13 @@ <h3>Multi-Post Storiese</h3>
               dummy text.
             </p>
             <div>
-              <div id="tb3">
-                <span id="a">HTML</span>
-                <span id="b">CSS</span>
-                <span id="c">BOOTSTRAP</span>
-                <span id="d">RUBY</span>
+              <div id="tb4">
+                <span id="a4">HTML</span>
+                <span id="b4">CSS</span>
+                <span id="c4">BOOTSTRAP</span>
+                <span id="d4">RUBY</span>
               </div>
```

Comment:
- [ ] `required`
- [ ] Here as mentioned above 
- [ ] However the list of project technologies are supposed to `ul` and each of them in a `li` see example `below`
![image](https://user-images.githubusercontent.com/66230306/186690799-18f92099-0f01-46bc-8bf3-46ea9828594f.png)
Kindly make amends on all the technologies